mlflow/server/js/src/components/ExperimentView.test.js

```diff
@@ -31,6 +31,8 @@ const getExperimentViewMock = () => {
     searchInput={""}
     searchRunsError={''}
     isLoading
+    orderByKey={null}
+    orderByAsc={false}
   />);
 };
 
```

Comment:
Add a new test that asserts the current implementation (that clicking "clear" calls the `onSearch` handler with proper arguments). In an ideal world we would actually test this in ExperimentPage.test.js as an integration test in which we deep-render ExperimentPage & its nested ExperimentView component (e.g. via [mount()](https://airbnb.io/enzyme/docs/api/mount.html)), call `onClear` in ExperimentView, and see props change in ExperimentPage. However it seems we can't currently do much better because our JS test runner doesn't actually support deep rendering, but that we might be able to add such support in the future (chatted about this with @Zangr). Open to not adding this test as it literally just asserts the current implementation.

```suggestion
test(`Clearing filter state calls search handler with correct arguments`, () => {
  const wrapper = getExperimentViewMock();
  const experiment = Fixtures.createExperiment();
  wrapper.instance().onClear();
  expect(onSearchSpy.mock.calls.length).toBe(1);
  expect(onSearchSpy.mock.calls[0][0]).toBe('');
  expect(onSearchSpy.mock.calls[0][1]).toBe('');
  expect(onSearchSpy.mock.calls[0][2]).toBe('');
  expect(onSearchSpy.mock.calls[0][3]).toBe(LIFECYCLE_FILTER.ACTIVE);
  expect(onSearchSpy.mock.calls[0][4]).toBe(null);
  expect(onSearchSpy.mock.calls[0][5]).toBe(null);
});
```